specs/protocol/isthmus/configurability.md

```diff
@@ -10,22 +10,34 @@
 - [`SystemConfig`](#systemconfig)
   - [`ConfigUpdate`](#configupdate)
   - [Initialization](#initialization)
+  - [Modifying Operator Fee Parameters](#modifying-operator-fee-parameters)
   - [Interface](#interface)
+    - [Operator fee parameters](#operator-fee-parameters)
+      - [`operatorFeeScalar`](#operatorfeescalar)
+      - [`operatorFeeConstant`](#operatorfeeconstant)
+      - [`setOperatorFeeScalars`](#setoperatorfeescalars)
+      - [`setOperatorFeeManager`](#setoperatorfeemanager)
```

Comment:
Can we create a role in the `SuperchainConfig` called "operator fee manager" and source the value from there? https://github.com/ethereum-optimism/specs/blob/d6b979ab67bd98bc3a15bf4988df08a066788047/specs/protocol/isthmus/superchain-config.md

This will make standard definition more simple, use the same superchain config that op gov recognizes, chain can also use their own superchain config if they are their own L2

Reply:
Follow up: trying to align ongoing work that is changing with this so that things are coherent, [this](https://github.com/ethereum-optimism/design-docs/blob/4980faff965e6caa1fc35ea015654c17c5931d10/protocol/standard-l2-genesis.md#systemconfig) is the design that we are landing on.

The `SystemConfig.initialize` function would accept a `Roles` struct, it would be good to add the `operatorFeeManager` to that `Roles` struct and have no setter function meaning the operator can only change with another call to `initialize`. This helps keep the ABI slim and allows for each chain operator to have its own individual account that is used.

It is still possible to have each chain have its own individual accounts in the world where its sourced from the `SuperchainConfig`, it just means that chains that aren't governed by Optimism would deploy their own `SuperchainConfig`. Apologies for the back and forth, was trying to be consistent with designs in the Standard L2 Genesis project and we were flip flopping internally on how we were going to do it

Reply:
This makes more sense to me than the superchain solution, happy to proceed in this direction. Once the spec lands for the `Roles` struct then I'll update this spec accordingly -- I've been yet unable to find reference to the design choices you linked.